<launch>
    <!-- Simulacao -->
    <include file="$(find butter_robot_large_base_urdf)/launch/butter.launch"></include>

    <!-- Behaviour -->
    <node name="behaviour" pkg="state_machine" type="robot.py" output="screen" >    </node>

    <!-- Vision -->
    <node name="vision" pkg="butter_detection" type="getting_frames_and_inferencing.py" output="screen" >    </node>
    <!--<node name="vision_sim" pkg="ballsim" type="Behaviour_test.py" output="screen" >    </node>-->

    <!-- Movement -->
    <node name="movement" pkg="robot_moves" type="simple_moves" output="screen" >    </node>

</launch>